### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by Redtigercod4**
This Pull Request adds the following:

1. Creates a system controller for all things related to the application
2. Allows the user to interrogate the endpoint for any load balancer requirements
3. Gives current uptime and request timestamp

### CODE REVIEW

File: api/src/services/SystemService.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export default class SystemService implements ISystemService {
+  public healthCheck(): HealthCheckType {
+    const uptimeTotalHours = Math.floor(process.uptime() / 3600);
+    const uptimeTotalMinutes = Math.floor((process.uptime() % 3600) / 60);
+    const uptimeTotalSeconds = Math.floor(process.uptime() % 60);
+    return {
+      uptime: `${uptimeTotalHours} hours : ${uptimeTotalMinutes} minutes : ${uptimeTotalSeconds} seconds`,
+      message: "OK",
+      timestamp: new Date(Date.now()),
```
Comment (SolarFactories) [inline api/src/services/SystemService.ts:15]: W: For simplicity keep `uptime` and `timestamp` as `number` of seconds, i.e. `process.uptime()` and `Date.now() / 1000`, with `Math.floor` if desired to have integers, making them more parseable by external systems.

File: api/src/utils/types/attributeTypes.ts [hunk 2]
```diff
@@ -51,19 +51,32 @@ export const enum Scope {
...
+    ? T
+    : T extends object
+    ? { [K in keyof T]?: JsonNetworkType<T[K]> }
+    : T extends Array<infer E>
+    ? Array<JsonNetworkType<E>>
+    : // Unknown case
+      T;
+
+export type HealthCheckType = {
+  uptime: any;
```
Comment (SolarFactories) [inline api/src/utils/types/attributeTypes.ts:79]: S: Refine from `any`, though if converting to `number` would be resolved.

Comment (Redtigercod4) [reply api/src/utils/types/attributeTypes.ts:79]: Yeah its the wrong type lol

**Review by SolarFactories — APPROVED** [review]
Once type of `uptime` within `HealthCheckType` is updated, is able to be merged.

File: api/src/utils/types/attributeTypes.ts (hunk 2 above)
Comment (Redtigercod4) [reply api/src/utils/types/attributeTypes.ts]: Commit: e5b664bca96f28bff5af1d3f474f78dadb02eafb

**Pull request merged**